TypeApprovalRegister+services.yaml

```diff
@@ -191,10 +238,12 @@ clients:
       application-name: AdministratorAdministration
       release-number: 1.0.0
       uuid: tar-1-0-0-http-c-0060
+            tar-1-0-0-http-c-bm-aa-1-0-0-000
```

Comment:
As per Structure of uuid definition. api-segment for AA http-c and tcp-c would be "bs", since we have only one op-c under EatL that too "bs"